lte/gateway/python/magma/mobilityd/mobility_store.py

```diff
@@ -71,8 +71,11 @@ class MacToIP(RedisFlatDict):
     """
     Used for managing DHCP state of a Mac address.
     """
-    def __init__(self):
-        client = get_default_client()
+    def __init__(self, client=None):
+        if client is None:
```

Comment:
since this will be updated later anyway, I'd suggest adding the TODO comment, but keep the implementation the same (the redis store class should not be checking against if the client is instantiated or not)